tests/expectations/tests/libclang-5/constant-evaluate.rs

```diff
@@ -12,7 +17,7 @@ pub enum _bindgen_ty_1 {
 }
 pub type EasyToOverflow = ::std::os::raw::c_ulonglong;
 pub const k: EasyToOverflow = 2147483648;
-pub const k_expr: EasyToOverflow = 0;
+pub const k_expr: EasyToOverflow = 1152921504606846976;
```

Comment:
It would be nice to have a test for the one that didn't even end up as a constant, but a static (`1 << 31`).

Reply:
Done. I'll have to update the older clang expectations for the test from travis, but that's only applying a diff once CI runs.